labgrid/driver/power/siglent.py

```diff
@@ -0,0 +1,21 @@
+""" tested with Siglent SPD3303X-E, and should be compatible with all SPD3000X series modules"""
```

Comment:
Maybe this should be called siglent_spd3000x.py to avoid confusion? You could also mention the "model" name in the changelog.

Reply:
I did consider that, but that would mean that `siglent_spd3000x` would be encoded in configuration files.
But as other Siglent PSU's are likely to be easy to support with the same code, such as SPD1000X series, I think it makes most sense to use a more generic name.
Only change I needed for other Siglent VXI-11 supplies would most likely be to adapt the check for index values, and probably also the decoding of status bits.

Reply:
Maybe siglent_spd or _vxi11 instead?

Reply:
I would go for siglent_vxi11 then.  And I should use `_` and not `-` ?

Reply:
`-` is not a valid character in python module names.

If you're confident that siglent won't switch, I'm fine with that. @Emantor?

Reply:
VXI-11 is based on HPIB/GPIB, which was defined in the late 1960's.  Instrument control is not an area with much new-shiny in the protocol development :grin:

Reply:
@jluebbe Is it okay to resolve this as it is now?